### PR TITLE
feat: add namespace default

### PR DESCRIPTION
**Pull request by Proximyst**
I would also prefer to set this on the app platform routes itself, but couldn't quickly find those.

### CODE REVIEW

File: pkg/registry/apis/provisioning/register.go [hunk 1]
```diff
@@ -447,5 +446,35 @@ spec:
...
+			paramsOrNil(sub.Head),
+			paramsOrNil(sub.Options),
+		} {
+			if params == nil {
+				continue
+			}
+
+			for _, p := range params {
+				if p.ParameterProps.Name == "namespace" && p.Schema.Default == nil {
+					p.Schema.Default = "default"
```
Comment (ryantxu) [inline pkg/registry/apis/provisioning/register.go:473]: the value really depends on the user... "default" is true on-prem when org=1

I tried to have the swagger UI fill it in automagicly:
https://github.com/grafana/grafana/blob/main/public/swagger/SwaggerPage.tsx#L65

it seems to work most of the time, but not in our branch -- i'll dig more tomorrow

Comment (Proximyst) [reply pkg/registry/apis/provisioning/register.go:473]: hmm! Would you be against doing this for now anyways, considering we're primarily targetting on-prem? :)

Comment (ryantxu) [reply pkg/registry/apis/provisioning/register.go:473]: yes -- think we can fix this so it shows the right thing.  Not having the right one in cloud blows.  For example:
https://play.grafana.org/swagger?api=banners.grafana.app-v0alpha1#/AnnouncementBanner/listNamespacedAnnouncementBanner

image

sucks when you don't know this is `stacks-35611`!

Comment (Proximyst) [reply pkg/registry/apis/provisioning/register.go:473]: that's fair! I'll deal with the bad default for now, then 😁 

I might take a look at the frontend soon, though I'm not huge on frontend code 😅

**Pull request closed without merging**